src/test/java/com/squareup/gifencoder/LzwEncoderTest.java

```diff
@@ -0,0 +1,21 @@
+package com.squareup.gifencoder;
+
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
```

Comment:
What an overwhelmingly specific assertion!

Reply:
I think the more recent AssertJ requires some Java 8 classes? So I changed to `Test(expected = ...)` :(